### ISSUE TITLE
Every new/already opened window becomes transparent

### ISSUE THREAD

**Issue by tranquilknight** [linuxmint/Cinnamon#6234]
See these screenshots: 
[http://imgur.com/a/TwXGH](url)
[http://imgur.com/a/6pnV1](url)

Open the full sized image and look closely (entire screen - BleachBit; middle - Screen locker). After using OS for a while every window becomes transparent. It happens at random time. I've tried to notice anything common before the occurrence of the event but I haven't noticed anything similar happening before each event.

If I open the menu, it appears normally. Anything after that becomes transparent. I have tried disabling effects. Problem remains. After logging out and logging in, everything becomes normal.

My laptop configuration is:
Mint 18.1 Cinnamon 64-bit (almost fresh install; installed a few programs and required repositories for ns3; tried to re-install WiFi driver but this problem existed before that)
i5 6200U
8 GB DDR4 RAM
2 GB AMD R5 M430/M330 (On Windows, before a couple of updates of graphics driver it was M430. After an update now it shows M330)

**Comment by leigh123linux** [comment]
Have you tried disabling the 'peek at desktop on hover' for the 'show desktop' applet?

**Comment by tranquilknight** [comment]
No, I haven't disabled that. However, I've tried clicking on the applet when windows became transparent. It didn't help in my case.

**Comment by MrElectroNick** [comment]
@leigh123linux @tranquilknight Same in my case. Only log out and log in again helps

**Comment by cruftytheclown** [comment]
For me it happens sometimes when I use the print screen key. Hitting print screen blacks out the entire screen for a split second and then the screen comes back (I guess that is intended). But, sometimes when it comes back everything is super transparent as if I am hovering over the show desktop icon and getting the "peek" view. First time I couldn't figure out what was happening and I had to reboot. But now it happened again, and for me clicking on Show Desktop fixes it.

So far I seem to be able to reproduce the error by having several images open in gimp and using print screen several times, but I'm not sure if it would always eventually do that, but it definitely is happening after I hit the Print Screen key.

**Comment by tranquilknight** [comment]
@cruftytheclown I did not press print screen key except taking screenshots of the bug. I'm away from that computer. I will try to reproduce the bug when I get a chance to use it.

**Comment by msadek** [comment]
Clicking on the Desktop button did resolve my case

**Comment by xcl706** [comment]
I have the same problem. Sometimes when i open a window, all window except for the panel area in the bottom of screen become transparent. I have to restart the mdm service.

There is my System Info:
System:    Host: xcl706-PAF4 Kernel: 4.10.0-27-generic x86_64 (64 bit gcc: 5.4.0)
                Desktop: Cinnamon 3.4.4 (Gtk 3.18.9-1ubuntu3.3) dm: mdm Distro: Linux Mint 18.2 Sonya
Machine: System: Hasee (portable) product: PAF4 v: 1.0
                Mobo: Hasee model: A24Win8 v: 1.0 Bios: American Megatrends v: 506 date: 04/25/2013
Info:         Processes: 287 Uptime: 35 min Memory: 2195.4/7859.9MB
                Init: systemd v: 229 runlevel: 5 default: 2 Gcc sys: 5.4.0

**Comment by katzio** [comment]
same, adding my system info as well. 
**Clicking on the Desktop button did resolve my case** @msadek Thanks!


System:    Host: feather Kernel: 4.10.0-38-generic x86_64 (64 bit gcc: 5.4.0)
           Desktop: Cinnamon 3.4.6 (Gtk 3.18.9-1ubuntu3.3) dm: lightdm Distro: Linux Mint 18.2 Sonya
Machine:   System: LENOVO (portable) product: 81A8 v: Lenovo ideapad 720S-13IKB
           Mobo: LENOVO model: LNVNB161216 v: SDK0J40709 WIN Bios: LENOVO v: 5SCN21WW date: 09/13/2017
           Chassis: type: 10 v: Lenovo ideapad 720S-13IKB
CPU:       Dual core Intel Core i5-7200U (-HT-MCP-) cache: 3072 KB
           flags: (lm nx sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx) bmips: 10848
           clock speeds: min/max: 400/3100 MHz 1: 600 MHz 2: 600 MHz 3: 596 MHz 4: 593 MHz
Graphics:  Card: Intel Device 5916 bus-ID: 00:02.0 chip-ID: 8086:5916
           Display Server: X.Org 1.18.4 drivers: intel (unloaded: fbdev,vesa)
           Resolution: 1920x1080@60.02hz
           GLX Renderer: Mesa DRI Intel HD Graphics 620 (Kabylake GT2)
           GLX Version: 3.0 Mesa 17.0.7 Direct Rendering: Yes
Audio:     Card Intel Device 9d71 driver: snd_hda_intel bus-ID: 00:1f.3 chip-ID: 8086:9d71

**Comment by Vahan86** [comment]
@tranquilknight, is this still an issue in Cinnamon 4.2?

**Comment by tranquilknight** [comment]
> 
> 
> @tranquilknight, is this still an issue in Cinnamon 4.2?

Not so far. Although I haven't used system with updated version that much.

**Comment by pedroolmedo98** [comment]
I have the same problem with Cinnamon 4.6.6 in Debian 10 bullseye/sid. New windows becomes semitransparent when i open a window at random times. Later, if i press the desktop button the problem is solved.
